lib/features/give/bloc/bloc.dart

```diff
@@ -1,2 +1,2 @@
+export '../../../shared/bloc/organisation/organisation.dart';
```

Comment:
relative paths or with the package format?

Reply:
🤔 lemme check